pkgs/development/python-modules/pyvips/default.nix

```
@@ -0,0 +1,38 @@
+{ buildPythonPackage, fetchPypi, python, pytestrunner, gcc, glib, cffi
+, python3Packages, pkgconfig, stdenv, vips, lib}:
```

pkg-config should be preferred over alias
```suggestion
, python3Packages, pkg-config, stdenv, vips, lib}:
```